libs/components/src/next/form/radio/vl-radio.component.ts

```diff
@@ -0,0 +1,122 @@
+import { CSSResult, html, PropertyDeclarations, TemplateResult } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import { live } from 'lit/directives/live.js';
+import { classMap } from 'lit/directives/class-map.js';
+import { radioStyle } from '@domg/govflanders-style/component';
+import { baseStyle, resetStyle } from '@domg/govflanders-style/common';
+import { vlElementsStyle } from '@domg-wc/elements';
+import radioUigStyle from './vl-radio.component.uig-css';
+import { BaseLitElement } from '@domg-wc/common-utilities';
+
+export const RadioDefaults = {
+    id: '',
+    value: '',
+    name: '',
+    block: false,
+    readonly: false,
+    disabled: false,
+    error: false,
+    success: false,
+    checked: false,
+};
+
+@customElement('vl-radio-next')
+export class VlRadioComponent extends BaseLitElement {
+    // Properties
+    private value = RadioDefaults.value;
+    private name = RadioDefaults.name;
+    private label: string = '';
+    private block = RadioDefaults.block;
+    private readonly = RadioDefaults.readonly;
+    private disabled = RadioDefaults.disabled;
+    private error = RadioDefaults.error;
+    private success = RadioDefaults.success;
+    private checked = RadioDefaults.checked;
+
+    static get styles(): (CSSResult | CSSResult[])[] {
+        return [resetStyle, baseStyle, vlElementsStyle, radioStyle, radioUigStyle];
+    }
+
+    static get properties(): PropertyDeclarations {
+        return {
+            value: { type: String },
+            name: { type: String },
+            label: { type: String },
+            disabled: { type: Boolean },
+            readonly: { type: Boolean },
+            error: { type: Boolean },
+            success: { type: Boolean },
+            block: { type: Boolean },
+            checked: { type: Boolean, reflect: true },
+        };
+    }
+
+    render(): TemplateResult {
+        const classes = {
+            'vl-radio': true,
+            'vl-radio--block': this.block,
+            'vl-radio--disabled': this.disabled,
+            'vl-radio--error': this.error,
+            'vl-radio--success': this.success,
+        };
+        return html`
+            <label id="radio-label" class=${classMap(classes)} for="radio">
+                <input
+                    class="vl-radio__toggle"
+                    type="radio"
+                    id="radio"
+                    .name=${this.name}
+                    aria-label=${this.label}
+                    ?aria-checked=${this.checked}
```

Comment:
Is dit nodig? Lijkt me dat HTML dit vanzelf zou afhandelen.

Reply:
Gezien het een custom component was dacht ik om het nog eens expliciet er op te zetten maar geeft geen problemen op vlak van accessibility tests als ik het verwijder > verwijderd!